tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Checks/IisCheckTests.cs

```diff
@@ -203,6 +203,42 @@ public async Task ListApplications()
             console.Output.Should().Contain(Resources.CouldNotFindIisApplication("sample", "/dummy"));
         }
 
+        [SkippableFact]
+        [Trait("RunOnWindows", "True")]
+        public async Task IncorrectlyConfiguredAppPool()
+        {
+            EnsureWindowsAndX64();
+
+            var buildPs1 = Path.Combine(EnvironmentTools.GetSolutionDirectory(), "tracer", "build.ps1");
+
+            EnvironmentHelper.SetAutomaticInstrumentation(false);
+
+            try
+            {
+                // GacFixture is not compatible with .NET Core, use the Nuke target instead
+                Process.Start("powershell", $"{buildPs1} GacAdd --framework net461").WaitForExit();
```

Comment:
If this is an aspnetcore all, we don't _need_ to add the libraries to the GAC do we? 🤔

Reply:
You are right, I removed it and confirmed the test passed in the CI, thank you!